numpy/core/tests/test_dtype.py

```diff
@@ -666,7 +667,7 @@ class user_def_subcls(np.void):
 class TestPickling(object):
 
     def check_pickling(self, dtype):
-        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
+        for proto in range(2, pickle.HIGHEST_PROTOCOL+1):
```

Comment:
You changed the style of the spacing around the operator for no reason. Please stay consistent with the existing code style.

Reply:
I'll go all over the `range(...)` iterations on the pickle protocol and change them all to one unique style.